docs/content/content/multilingual.md

```diff
@@ -0,0 +1,155 @@
+---
+date: 2016-01-02T21:21:00Z
+menu:
+  main:
+    parent: content
+next: /content/example
+prev: /content/summaries
+title: Multilingual Mode
+weight: 68
+toc: true
+---
+
+Since version 0.16, Hugo supports a native Multilingual mode. You
+define the languages to render as such:
+
+```
+Multilingual:
+  en:
+    weight: 1
+    title: "My blog"
+    params:
+      linkedin: "english-link"
+  fr:
+    weight: 2
+
+    title: "Mon blog"
+    params:
+      linkedin: "lien-francais"
+    copyright: "Tout est miens"
+
+copyright: "Everything is mine"
+```
+
+Anything not defined in a `lang:` block will fall back to the global
+value for that key (like `copyright` in this example).
+
+With the config above, all content, sitemap, RSS feeds, paginations
+and taxonomy pages will be rendered under `/en` in English, and under
+`/fr` in French.
+
+
+### Translating your content
+
+Translated articles are picked up by the name of the content files.
+
+Example of translated articles:
+
+1. `/content/about.en.md`
+2. `/content/about.fr.md`
+
+You can also have:
+
+1. `/content/about.md`
+2. `/content/about.fr.md`
+
+in which case the config variable `DefaultContentLanguage` will be
+used to affect the default language `about.md`.  This way, you can
+slowly start to translate your current content without having to
+rename everything.
+
+If left unspecified, the value for `DefaultContentLanguage` defaults
+to `en`.
+
+By having the same _base file name_, the content pieces are linked
+together as translated pieces. Only the content pieces in the language
+defined by **.Site.CurrentLanguage** will be rendered in a run of
+`hugo`.  The translated content will be available in the
+`.Page.Translations` so you can create links to the corresponding
+translated pieces.
+
+
+### Language switching links
+
+A full example of the language-switching links would be:
```

Comment:
yes, ok I agree.. I started at the beginning with `.Translations` excluding the current language, and I don't remember the reason why I added the current translations (I think it might have had to do with the fact that otherwise there's no place where you can access a complete picture of all the pages on a site.).

I agree with reverting that to the behavior you propose.  It will indeed simply the example (get rid of `{{ if ne $lang $txLang}}).... we'll still need `{{ if isset $.Translations $txLang }}` and the rest.. so it'll remove 4 lines in the example.

Do you see other ways we can shrink that ?